fhirResourcesToLoad/Practitioner-Fairchild.json

```diff
@@ -0,0 +1,16 @@
+{
+    "resourceType": "Practitioner",
+    "id": "pra-hfairchild",
```

Comment:
Need Practitioner.identifier

Reply:
Is it required? According to https://www.hl7.org/fhir/practitioner.html, the cardinality is from 0...*.

Reply:
We are using US Core profile if possible: http://hl7.org/fhir/us/core/StructureDefinition-us-core-practitioner.html